{"source": "exploit-database", "filename": "platforms/multiple/dos/41145.py", "cves": ["CVE-2017-3241"]}